lib/utils.js

```diff
@@ -53,20 +53,25 @@ function appender(xs) {
     return xs
   }
 }
-
+function padStart(number, expectedLength) {
+  const strNumber = String(Math.abs(number)).padStart(expectedLength, '0')
+  return (number < 0) ? `-${strNumber}` : strNumber
+}
```

Comment:
This can be reduced to (given that times are not negative):

```js
function padStart(number, len) {
  return `${number}`.padStart(len, '0')
}
```